mopro-ffi/src/lib.rs

```diff
@@ -233,28 +353,38 @@ macro_rules! app {
                     a: mopro_ffi::G1 {
                         x: self.a.x,
                         y: self.a.y,
+                        z: None,
                     },
                     b: mopro_ffi::G2 {
                         x: self.b.x,
                         y: self.b.y,
+                        z: None,
                     },
                     c: mopro_ffi::G1 {
                         x: self.c.x,
                         y: self.c.y,
+                        z: None,
                     },
                 }
             }
         }
+        // End of Ethereum calldata Section
 
         #[derive(Debug, Clone, Default, uniffi::Enum)]
         pub enum ProofLib {
             #[default]
             Arkworks,
-            Rapidsnark,
+            RapidSnark,
```

Comment:
actually I think we don't need to capitalize `S` in `RapidSnark`
https://github.com/iden3/rapidsnark?tab=readme-ov-file#rapidsnark
when it is mentioned, only `R` is capitalized

Reply:
I just tried to align what we have in [circom-prover](https://github.com/zkmopro/mopro/blob/main/circom-prover/src/prover.rs#L25). But I unified the naming in fixed in [a0551c9](https://github.com/zkmopro/mopro/pull/372/commits/a0551c983b38309cd18d6e21617883a74b541b1f)